<?xml version="1.0"?>

<launch>
    <node name="republish" type="republish" pkg="image_transport" output="screen" args="compressed in:=/zed2/zed_node/rgb/image_rect_color raw out:=/zed2/zed_node/rgb/image_rect_color" />
    <node name="image_preprocessing_node"    pkg="image_preprocessing_cpp"     type="image_preprocessing_cpp"    output="screen"/>
    <rosparam command="load" file="$(find image_preprocessing_cpp)/params/preprocessing.yaml" />
</launch>
